pandas/core/frame.py

```diff
@@ -2868,8 +2865,6 @@ def _ixs(self, i: int, axis: int = 0):
         if axis == 0:
             label = self.index[i]
             new_values = self._data.fast_xs(i)
-            if is_scalar(new_values):
```

Comment:
this is a perf optimization IIRC, can you check

Reply:
Its never hit.  I'm fairly confident that it was there because of a now-fixed bug in DatetimeTZBlock.get that incorrectly returned scalar instead of 1D